app/src/main/java/de/westnordost/streetcomplete/ApplicationConstants.kt

```diff
@@ -63,4 +65,8 @@ object ApplicationConstants {
         // no wiki entry, sounds like it could span large areas
         "power", "pipeline", "railway"
     )
+
+    val EDIT_ACTIONS_NOT_ALLOWED_TO_USE_LOCAL_CHANGES = setOf(
+        SplitWayAction::class
```

Comment:
```suggestion
        /* because this action may edit route relations but route relations are not persisted 
           locally for performance reasons */
        SplitWayAction::class
```